src/Microsoft.AspNetCore.Http.Features/IHttpRequestFeature.cs

```diff
@@ -49,6 +49,11 @@ public interface IHttpRequestFeature
         string QueryString { get; set; }
 
         /// <summary>
+        /// The request target as it was sent in the HTTP request.
+        /// </summary>
+        string RawTarget { get; set; }
```

Comment:
The problem with `RawPath` is that it's incorrect. We'll include the query with it, no just the path. Plus the feature distinguishes between `Path` and `PathBase`, which makes it even more confusing.